package.json

```diff
@@ -122,10 +122,10 @@
     "@rollup/plugin-node-resolve": "^14.1.0",
     "@types/css-tree": "^2.0.0",
     "@types/csso": "^5.0.0",
-    "@types/jest": "^29.1.1",
+    "@types/jest": "^29.5.5",
     "del": "^6.0.0",
     "eslint": "^8.24.0",
-    "jest": "^29.1.2",
+    "jest": "^29.5.5",
```

Comment:
You can import from `@jest/globals` to get types instead of `@types/jest`

Reply:
It's unclear to me how to use `@jest/globals` properly when the package isn't declared as a module. When I use the `require`, nothing works as expected, and I can't import the types with `@typedef`. 🤔 

Despite this, the tests do pass, so the library works. I'm just struggling with the TypeScript integration.

Reply:
Ah, right. Forgot we don't use ts syntax.